components/Buttons/src/MDCButton.h

```diff
@@ -20,6 +20,8 @@
 #import "MaterialInk.h"
 #import "MaterialShadowElevations.h"
 
+@protocol MDCShapeGenerating;
```

Comment:
Import the shapes component that exposes this API instead please - we're still generally trying to avoid forward declarations of symbols not contained within a given component.

Reply:
done